src/translations/en.json

```diff
@@ -1506,6 +1506,20 @@
             "create": "Create"
           }
         },
+        "mqtt": {
```

Comment:
These will need to be renamed in Lokalise.

Reply:
Done.